### ISSUE TITLE
[Accessibility]: Duplicate headings for va-accordion-item in review & submit pattern

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/vets-design-system-documentation#5209]
### Point of contact

Jamie Klenetsky Fay

### How severe do you think this issue is?

High

### Severity rationale

These are duplicate headings. If you use the rotor menu to navigate by heading, you won't know which one to select, and only one of the two will take you to the interactive button needed to open the accordion. While it's _possible_ to navigate through, it is a frustrating experience.

### URL where issue occurs

https://staging.va.gov/family-and-caregiver-benefits/survivor-compensation/apply-for-accrued-benefits-form-21p-601/review-and-submit

### Component or pattern (if applicable)

va-accordion-item in the review & submit component

### Environment details

- OS: MacOS 26.1
- Browser: Chrome
- Screen reader: VoiceOver

### Steps to reproduce

1. Log in as a test user
2. Fill out form until you get to the Review application page (last page)
3. Use [HeadingsMap](https://chromewebstore.google.com/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) or the VoiceOver rotor menu to view a list of headings on the page

### Observed behavior

HeadingsMap shows nested headings:

<img width="596" height="394" alt="Image" src="https://github.com/user-attachments/assets/75dd3ed2-ef16-4aac-9851-5d1329444f41" />

The VO rotor menu shows duplicate headings:

<img width="621" height="858" alt="Image" src="https://github.com/user-attachments/assets/dcbbae4b-7977-48a9-8e13-6969a03eb36c" />

Interestingly, the JAWS heading menu doesn't seem to pick up the duplicates:

<img width="346" height="188" alt="Image" src="https://github.com/user-attachments/assets/c4e0df12-319c-488e-8273-b3d0267df7a2" />

### Expected behavior / Acceptance criteria

Each va-accordion-item should have only 1 heading to select to open the accordion.

### Supporting evidence (screenshots, recordings, code snippets)

Jeana and I think this is specific to the review & submit pattern in the forms library because this issue doesn't occur:
- in [Storybook](https://design.va.gov/storybook/iframe.html?globals=&args=&id=uswds-va-accordion--default&viewMode=story)
- in other uses of va-accordion-item ([example](https://staging.va.gov/supporting-forms-for-claims/alternate-signer-form-21-0972/confirmation))

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [ ] Focus
- [x] Headings
- [x] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else
